apis/frontend/oss/README.md

```diff
@@ -5,8 +5,8 @@ The libraries are hosted on GitHub.
 
 | Library | Description | Links |
 | :-- | :-- | :-- |
-| [nosto-js](https://docs.nosto.com/techdocs/apis/frontend/oss/nosto-js) | Nosto JS is a wrapper for the client script, providing an easy way to interact with the Nosto service from your JavaScript or TypeScript applications. Use this library for basic Nosto integration in FE app builds. | [GitHub](https://github.com/Nosto/nosto-js) <br/> [API Docs](https://nosto.github.io/nosto-js/) |
-| [search-js](https://docs.nosto.com/techdocs/apis/frontend/oss/search-js) | Search JS is a wrapper for the Nosto Search functionality. Use this library for Nosto Search usage in FE app builds. | [GitHub](https://github.com/Nosto/search-js) <br/> [API Docs](https://nosto.github.io/search-js/) |
-| [nosto-autocomplete](https://docs.nosto.com/techdocs/implementing-nosto/implement-search/search/implement-autocomplete-using-library) | Standalone library for Autocomplete implementations using Nosto Search. Use this library for Autocomplete implementations using Nosto Search in FE app builds. | [GitHub](https://github.com/Nosto/nosto-autocomplete) <br/> [API Docs](https://nosto.github.io/nosto-autocomplete) |
-| [nosto-react](https://docs.nosto.com/techdocs/apis/frontend/oss/react-support) | Nosto React is a React component library to make it easier to implement Nosto using the Session API. Use this library for embedding Nosto recommendations in a React app. | [GitHub](https://github.com/Nosto/nosto-react) <br/> [API Docs](https://nosto.github.io/nosto-react/) |
+| [nosto-js](../nosto-js) | Nosto JS is a wrapper for the client script, providing an easy way to interact with the Nosto service from your JavaScript or TypeScript applications. Use this library for basic Nosto integration in FE app builds. | [GitHub](https://github.com/Nosto/nosto-js) <br/> [API Docs](https://nosto.github.io/nosto-js/) |
+| [search-js](../search-js) | Search JS is a wrapper for the Nosto Search functionality. Use this library for Nosto Search usage in FE app builds. | [GitHub](https://github.com/Nosto/search-js) <br/> [API Docs](https://nosto.github.io/search-js/) |
+| [nosto-autocomplete](../../../implementing-nosto/implement-search/search/implement-autocomplete-using-library) | Standalone library for Autocomplete implementations using Nosto Search. Use this library for Autocomplete implementations using Nosto Search in FE app builds. | [GitHub](https://github.com/Nosto/nosto-autocomplete) <br/> [API Docs](https://nosto.github.io/nosto-autocomplete) |
```

Comment:
With this approach, the linking of docs from a different section in future could get more complex. Any disadvantages with using the full URL?

Reply:
page reloads